# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/xhr/XMLHttpRequest.h
# Commit: 51bd2ff91c1a
# Full Hash: 51bd2ff91c1a4fd7f12dc5b4062be8657e82f8ee
# Author: Thomas Wisniewski <twisniewski@mozilla.com>
# Date: 2024-06-21 10:09:55
# Regressor Bug: 1842970
# File Overlap Count: 1
# Description:
#   Bug 1842970 - fire abort events more in line with the spec for worker XMLHttpRequest; r=valentin,sunil,dom-worker-reviewers,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D201744
# ==============================================================================

diff -r f8d683b59147 -r 51bd2ff91c1a dom/xhr/XMLHttpRequest.h
--- a/dom/xhr/XMLHttpRequest.h	Fri Jun 21 06:11:46 2024 +0300
+++ b/dom/xhr/XMLHttpRequest.h	Fri Jun 21 04:16:11 2024 +0000
@@ -76,7 +76,7 @@
   static constexpr ProgressEventType NAME { #NAME, u## #NAME }
 
 #define DECL_ERRORPROGRESSEVENT(NAME, ERR) \
-  static constexpr ErrorProgressEventType NAME{#NAME, u## #NAME, ERR};
+  static constexpr ErrorProgressEventType NAME { #NAME, u## #NAME, ERR };
 
   struct Events;
 